skema/text_reading/scala/src/main/scala/org/ml4ai/skema/text_reading/serializer/SkemaJSONSerializer.scala

```diff
@@ -416,7 +416,7 @@ object SkemaJSONSerializer {
       val argHashes = for {
         (role, mns) <- args
         bh = stringHash(s"role:$role")
-        hs = mns.map(_.equivalenceHash)
+        hs = mns.map(MentionOps(_).equivalenceHash)
```

Comment:
This can also cause problems with unknown mention types.  The solution is to write a SkemaMentionOps which accounts for any new kinds of Mentions that Skema introduces (if they could be among the args).